src/components/config-pages/local-footer.tsx

```diff
@@ -49,7 +49,7 @@ const LocalFooter = async () => {
   }
 
   return (
-    <div className="local-footer bg-foggy-light py-20">
+    <div className="local-footer border-4 border-white bg-foggy-light py-20">
```

Comment:
This adds a bottom and left/right borders. This seems contrary to our normal designs. Is this accurate? Or it could be on the other footer
![image](https://github.com/user-attachments/assets/0bbeee00-0bd2-4b79-975b-026b634ba635)

Reply:
Kat and I discussed this in depth today. She would like the brand bar, header, footer, and university footer to have a 4px border around them (originally it was just the header and footer but the mocks didn't include the brand bar and university footer). 🤔

Reply:
Did you show this to her? I think talking about something is one thing, but then seeing it is another. Personally, I do not care for it in the slightest bit. It makes the site look unfinished, like there are gaps in the styling.

Reply:
She mocked it up in figma [here](https://www.figma.com/design/vnc2oxvzX9qZ3rK5vmjpdZ/Summer-Session-Wireframes?node-id=4846-40842&t=cWk094byARC7DlGd-1) and [here](https://www.figma.com/design/vnc2oxvzX9qZ3rK5vmjpdZ/Summer-Session-Wireframes?node-id=2966-1286&t=cWk094byARC7DlGd-1) 😄

Reply:
Let's show the preview url to her and possibly to the client. I don't believe this will be a popular style choice.

Reply:
From Summer Session Sprint Planning: Drop white borders for brand bar, header, footer, and global footer